map/src/main/kotlin/edu/artic/map/MarkerGenerators.kt

```diff
@@ -0,0 +1,139 @@
+package edu.artic.map
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+
+/**
+ * Description:
```

Comment:
Drop the first comment block, it's not doing anything much.

Reply:
oops great catch! ha.